src/BloomExe/CollectionChoosing/OpenCreateCloneControl.cs

```diff
@@ -201,6 +204,58 @@ public void SelectCollectionAndClose(string path)
 			}
 		}
 
+		/// <summary>
+		/// Path to the basic user configuration information file for Dropbox.  It may or may not exist.
+		/// We make it a static variable because we only want to work at getting the Dropbox folder once
+		/// and this make a suitable flag for doing that.
+		/// </summary>
+		private static string _dropboxInfoFile;
+		/// <summary>
+		/// Path to the user's Dropbox folder.  It is static because we only want to look it up once.
+		/// </summary>
+		private static string _dropboxFolder;
+		/// <summary>
+		/// This method checks 'path' for being in the Dropbox folder.  If so, it displays a warning
+		/// message and returns true.
+		/// </summary>
+		/// <returns>true if 'path' is in the Dropbox folder</returns>
+		internal static bool CheckForBeingInDropboxFolder(string path)
+		{
+			if (_dropboxInfoFile == null)
+			{
+				var baseFolder = Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData);
```

Comment:
baseFolder gets set to, eg, C:\Users\mcconnel\AppData\Roaming.

LocalApplicationDataFolder would point to C:\Users\mcconnel\AppData\Local.